frontend/src/metabase/api/timeline-event.ts

```diff
@@ -46,7 +45,7 @@ export const timelineEventApi = Api.injectEndpoints({
         invalidateTags(error, [
           listTag("timeline-event"),
           idTag("timeline-event", id),
```

Comment:
I think not. Currently it would lead to unexpected results. E.g. you would invalidate a `collection` because it's referenced. I think these helpers can only be used in `providesTags`.

Reply:
Btw, maybe it should be reflected in the name. Let me try it out

Reply:
Added `provide` prefix for all the new helpers to identify that they can only be used in `providesTags`. Wdyt?

Reply:
Great idea!
Currently the only way to enforce it is via code reviews, so we should probably document it somewhere (and announce it).
Maybe a `README.md` in `metabase/api`?